data/json/monstergroups.json

```diff
@@ -552,30 +553,24 @@
       { "monster" : "mon_zombie_grabber", "freq" : 35, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_grappler", "freq" : 5, "cost_multiplier" : 10 },
       { "monster" : "mon_zombie_hunter", "freq" : 10, "cost_multiplier" : 5 },
-      { "monster" : "mon_skeleton", "freq" : 30, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_smoker", "freq" : 10, "cost_multiplier" : 5 },
+      { "monster" : "mon_zombie_smoker", "freq" : 10, "cost_multiplier" : 5, "starts" :   120 },
       { "monster" : "mon_zombie_shady", "freq" : 10, "cost_multiplier" : 4},
       { "monster" : "mon_zombie_shady", "freq" : 30, "cost_multiplier" : 15, "pack_size": [4,6]},
       { "monster" : "mon_zombie_gasbag", "freq" : 10, "cost_multiplier": 5},
       { "monster" : "mon_zombie_gasbag", "freq" : 3, "cost_multiplier": 10, "pack_size": [3,5]},
       { "monster" : "mon_zombie_swimmer", "freq" : 10, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_shrieker", "freq" : 30, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_spitter", "freq" : 20, "cost_multiplier" : 5 },
+      { "monster" : "mon_zombie_spitter", "freq" : 20, "cost_multiplier" : 5, "starts" :   120 },
       { "monster" : "mon_zombie_acidic", "freq" : 30, "cost_multiplier" : 2 },
       { "monster" : "mon_zombie_electric", "freq" : 10, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_necro", "freq" : 5, "cost_multiplier" : 25 },
       { "monster" : "mon_zombie_survivor", "freq" : 1, "cost_multiplier" : 25 },
       { "monster" : "mon_boomer", "freq" : 30, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_brute", "freq" : 10, "cost_multiplier" : 15 },
-      { "monster" : "mon_zombie_hulk", "freq" : 1, "cost_multiplier" : 50 },
-      { "monster" : "mon_skeleton_hulk", "freq" : 1, "cost_multiplier" : 50 },
-      { "monster" : "mon_zombie_master", "freq" : 1, "cost_multiplier" : 30 },
       { "monster" : "mon_beekeeper", "freq" : 1, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_technician", "freq" : 1, "cost_multiplier" : 12},
-      { "monster" : "mon_zombie_brute_shocker", "freq" : 5, "cost_multiplier" : 40 },
       { "monster" : "mon_zombie_runner", "freq" : 20, "cost_multiplier" : 5, "pack_size": [1,4] }
     ],
-    "replace_monster_group" : true,
+    "replace_monster_group" : false,
```

Comment:
Why turn this off?

Reply:
Good spot, that one snuck in from a testing version, fixing